arm-web/2015-08-01/swagger/AppServiceCertificateOrders.json

```diff
@@ -821,47 +821,36 @@
     "AppServiceCertificate": {
       "description": "Key Vault container for a certificate that is purchased through Azure.",
       "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/Resource"
-        }
-      ],
       "properties": {
-        "properties": {
-          "description": "ARM envelope properties",
-          "properties": {
-            "keyVaultId": {
-              "description": "Key Vault Csm resource Id.",
-              "type": "string"
-            },
-            "keyVaultSecretName": {
-              "description": "Key Vault secret name.",
-              "type": "string"
-            },
-            "provisioningState": {
-              "description": "Status of the Key Vault secret.",
-              "enum": [
-                "Initialized",
-                "WaitingOnCertificateOrder",
-                "Succeeded",
-                "CertificateOrderFailed",
-                "OperationNotPermittedOnKeyVault",
-                "AzureServiceUnauthorizedToAccessKeyVault",
-                "KeyVaultDoesNotExist",
-                "KeyVaultSecretDoesNotExist",
-                "UnknownError",
-                "ExternalPrivateKey",
-                "Unknown"
-              ],
-              "type": "string",
-              "readOnly": true,
-              "x-ms-enum": {
-                "name": "KeyVaultSecretStatus",
-                "modelAsString": false
-              }
-            }
-          },
-          "x-ms-client-flatten": true
+        "keyVaultId": {
+          "description": "Key Vault resource Id.",
+          "type": "string"
+        },
+        "keyVaultSecretName": {
+          "description": "Key Vault secret name.",
+          "type": "string"
+        },
+        "provisioningState": {
+          "description": "Status of the Key Vault secret.",
+          "enum": [
+            "Initialized",
+            "WaitingOnCertificateOrder",
+            "Succeeded",
+            "CertificateOrderFailed",
+            "OperationNotPermittedOnKeyVault",
+            "AzureServiceUnauthorizedToAccessKeyVault",
+            "KeyVaultDoesNotExist",
+            "KeyVaultSecretDoesNotExist",
+            "UnknownError",
+            "ExternalPrivateKey",
+            "Unknown"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "KeyVaultSecretStatus",
+            "modelAsString": false
```

Comment:
Setting modelAsString to false can result in breaking changes whenever you modify this enum